front_end/html/edit_exercise.html

```diff
@@ -489,21 +489,24 @@ <h2>Create exercise</h2>
             }
 
           $('div[contenteditable]').bind('paste', function(e) {
```

Comment:
This same code is used in multiple locations. I forget where it is, but I believe we have a file in CodeBuddy that includes Javascript functions that are shared across all pages. Please find a way to move the code into a function and put it in that file and then invoke the function from here.